distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -558,6 +581,16 @@ type Vector
         eq_at i = this.unsafe_at i == that.unsafe_at i
         if this.length == that.length then 0.up_to this.length . all eq_at else False
 
+    ## PRIVATE
+       Copy the backing Array to an Enso Array
+       TODO This is a workaround for the issue with polyglot Array.copy
+            https://www.pivotaltracker.com/story/show/181122542
+    copy_array : Array
+    copy_array start_index dest_array dest_start length =
+        case Builtins.Meta.is_polyglot this.to_array of
+            True -> 0.up_to length . each i->(dest_array.set_at (i+dest_start) (this.to_array.at i+start_index))
+            False -> Array.copy this.to_array start_index dest_array dest_start length
```

Comment:
This is quite an internal function **and** it does not really rely on being a member of Vector - it's not really a Vector operation, it's an operation on Arrays.

I would thus suggest to make it 'static', i.e.
`copy_array src_array src_start dest_array dest_start length = ...`